package.json

```diff
@@ -176,7 +176,7 @@
     "react-test-renderer": "16.12.0",
     "**/react-test-renderer": "16.12.0",
     "**/sharp": "0.23.3",
-    "cypress": "3.8.3"
+    "cypress": "4.0.1"
```

Comment:
This should allow us to update in the MC without a resolution /cc @ahmehri

Reply:
FYI: this dependency does not have anything to do with the one we use in the MC. Or am I missing something?

Reply:
@tdeekens meant to point out a tip for me to be able to update cypress to v4 in mc. Thanks.

Reply:
If you install two versions of cypress without a resolution your download of cypress seems to die.